rest-servlet/src/main/java/org/hawkular/inventory/rest/RestTenants.java

```diff
@@ -100,4 +134,20 @@ public Response delete() {
         return Response.noContent().build();
     }
 
+    @DELETE
+    @Path("/")
+    @ApiOperation("Deletes the tenant and all its data. Be careful!")
+    @ApiResponses({
+                          @ApiResponse(code = 204, message = "OK"),
+                          @ApiResponse(code = 401, message = "Unauthorized access"),
+                          @ApiResponse(code = 404, message = "Tenant doesn't exist", response = ApiError.class),
+                          @ApiResponse(code = 500, message = "Server error", response = ApiError.class)
+                  })
+    public Response delete(@PathParam("tenantId") String tenantId) {
```

Comment:
right, I missed that, the parameter is not used at all

> Also for consistency with GET /tenants/{tenantId}, should this be @Path("/tenants/{tenantId}")?
I wanted the URI in the relationship json to be the real URL, but perhaps it's not a good idea, because it's not only the tenants.. All the endpoints would need to have some kind of shadow implementation to be responding also for the old form of the api like:
* `28026b36-8fe4-4332-84c8-524e173a68bf/metricTypes/status.code.type`
* `28026b36-8fe4-4332-84c8-524e173a68bf/resourceTypes/URL`
* etc.

Using the URI without the tenantId is not good idea either, because of the uniqueness, perhaps the not meaningful URIs are just fine. Not very HATEOAS, but better than adding more complexity to the rest classes.